src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -324,26 +340,50 @@ private void validateCvrFileSources() {
     }
   }
 
+  private boolean candidateStringIsAlreadyInUseElsewhere(
```

Comment:
Please add a line or two comment to describe what this does. The parameter names are a bit confusing.